### PR TITLE
Make scripts easy to copy and read 

### PR DESCRIPTION
**Pull request by resouer**
As the title said :) Very small fix.

### CODE REVIEW

File: docs/getting-started-guides/docker-multinode/master.sh [hunk 1]
```diff
@@ -98,19 +106,40 @@ DOCKER_CONF=""
 
 start_k8s(){
     # Start etcd 
-    docker -H unix:///var/run/docker-bootstrap.sock run --restart=always --net=host -d gcr.io/google_containers/etcd:2.2.1 /usr/local/bin/etcd --addr=127.0.0.1:4001 --bind-addr=0.0.0.0:4001 --data-dir=/var/etcd/data
+    docker -H unix:///var/run/docker-bootstrap.sock run \
+    --restart=always \
+    --net=host \
+    -d \
```
Comment (ashcrow) [inline docs/getting-started-guides/docker-multinode/master.sh]: Why leave the value off the switch for this line?

**Comment by ashcrow** [comment]
@k8s-teamcity-mesosphere failures are flakes.

This looks good to me!

**Comment by wojtek-t** [comment]
@resouer - can you please squash the commits?

**Comment by resouer** [comment]
@wojtek-t  Yes. done!

**Comment by wojtek-t** [comment]
@k8s-bot test this please

**Pull request merged**